plugin.json

```diff
@@ -288,9 +288,9 @@
     ]
   },
   "props": {
-    "min_rtcd_version": "v0.12.0",
+    "min_rtcd_version": "v0.14.0",
     "min_offloader_version": "v0.7.0",
```

Comment:
Two things: 

- We don't need to bump `rtcd`. I actually wouldn't do that.
- May want to bump offloader though given config validation will be outdated otherwise.

Reply:
@streamer45 Ok, but why not update rtcd?

Reply:
Mostly because there's no need to. I see updating rtcd here same as updating `min_server_version`. We can do it but keeping some compatibility is a bit easier on upgrades otherwise we can get into sticky situations like when upgrading Community and Hub nothing works unless rtcd is upgraded before the plugin.

Reply:
Ok, sounds good